### DISCUSSION TITLE
When will all of the pylint rules be supported?

### DISCUSSION THREAD

**Discussion by ughstudios** [astral-sh/ruff#3406] [Q&A]
Is there any intention to reach parity with pylints rulset? It seems like over 200 rules are missing.

**Comment by JacobCoffee** [comment]
It is being worked on as best as I can remember.

Parity tracking issue: #970 

Docs describing this some: https://beta.ruff.rs/docs/faq/#how-does-ruff-compare-to-pylint
> At time of writing, Pylint implements ~409 total rules, while Ruff implements 440, of which at least 89 overlap with the Pylint rule set 

But as you see from #970, it awaits people to pick up the issues or @charliermarsh to spend some time on it so it may be a little while before we are closer 😬
